### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by tcluri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 # parktain
 A slack bot for the park-ers
 
+## Before you install
+
+Create a virtualenv and activate it(recommended):
+
+    virtualenv mydir
+    cd mydir
+    source activate/bin
```
Comment (punchagan) [inline README.md]: Combine this into `source mydir/activate/bin`

File: README.md [hunk 2]
```diff
@@ -15,3 +32,23 @@ To install the bot:
...
+
+Paste the following as is into config.yaml(or create it):
+
+    gendo:
+      auth_token: "API-token-from-previous-step"
+
+## Run
+
+To run your bot:
+    python main.py
```
Comment (punchagan) [inline README.md:56]: Needs a blank line to be rendered correctly.

**Comment by punchagan** [comment]
Looks good! Thanks @tcluri

**Comment by tcluri** [comment]
Thank you @punchagan :)

**Pull request merged**